projects/storefrontlib/shared/components/anonymous-consents-dialog/anonymous-consent-dialog.component.ts

```diff
@@ -214,6 +243,10 @@ export class AnonymousConsentDialogComponent implements OnInit, OnDestroy {
     }
   }
 
+  closeMessage(): void {
```

Comment:
Lets add a quick unit test for this method since it isn't covered afaik

Reply:
Done